web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -96,7 +96,7 @@ describe("<SidebarCardsList />", () => {
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
 
-  describe("empty state messaging", () => {
+  describe("Empty State Messaging", () => {
```

Comment:
Not part of the story, but the lowercase bothered my eyes. Since I worked on this, I figured it'd be ok to update here.